history/cn_creation.sql

```diff
@@ -0,0 +1,33 @@
+BEGIN TRANSACTION;
+DROP TABLE IF EXISTS "RULES";
+CREATE TABLE IF NOT EXISTS "RULES" (
+	"ID"		INTEGER NOT NULL UNIQUE,
+	"TIME_MS"	INTEGER NOT NULL DEFAULT 50,
+	"ROUNDS"	INTEGER NOT NULL DEFAULT 300,
+	PRIMARY KEY("ID" AUTOINCREMENT)
+);
+DROP TABLE IF EXISTS "GAMES";
+CREATE TABLE IF NOT EXISTS "GAMES" (
+	"ID"		INTEGER NOT NULL UNIQUE,
+	"DATE"		TEXT NOT NULL,
+	"BOT_A"		INTEGER NOT NULL,
+	"BOT_B"		INTEGER NOT NULL,
+	"POINTS_A"	INTEGER NOT NULL,
+	"POINTS_B"	INTEGER NOT NULL,
+	"RULES"		INTEGER NOT NULL,
+	"RECORD"	BLOB NOT NULL,
+	FOREIGN KEY("BOT_A") REFERENCES "BOTS"("ID"),
+	FOREIGN KEY("RULES") REFERENCES "RULES"("ID"),
+	FOREIGN KEY("BOT_B") REFERENCES "BOTS"("ID"),
```

Comment:
Did you try to run this script on empty DB ?
I'm afraid that you will not be able to create references to BOTS table before you create it  :)

Reply:
Yes. I ran the script on an empty database and everything works. I'm changing it.